Automattic/pocket-casts-android
# What's new bookmarks UI

## Description
This adds basic UI and navigation for `What's new `for bookmarks.

It does not yet update conditions for display and dynamic titles/ messages/ buttons.

## Testing Instructions
1. Fresh install the app 
2. Increment `Settings.WHATS_NEW_VERSION_CODE`
3. Re-install the app
4. ✅ Notice that `What's new` for `Bookmarks` is shown 
5. Tap on close button on top right
6. ✅ Notice that the dialog closes
7. Repeat steps 1-4
8. Tap on `Try it now`
9. ✅ Notice that headphone controls settings screen is opened

## Screenshots or Screencast 

<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/740b484a-2049-4b50-a749-3a07cbcb000a"/>


## Checklist
 
#### I have tested any UI changes...
